File: 1-two-sum/1-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        temp = []
        for i in range(0, len(nums)):
            if (target - nums[i]) in temp:
                return [temp.index(target - nums[i]), i]
            temp.append(nums[i])

File: 1-two-sum/1-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int ans[] = new int[2];
        HashMap<Integer,Integer> temp = new HashMap<>();
        for(int i=0; i<nums.length;i++) {
            int diff = target - nums[i];
            if(temp.containsKey(diff)) {
                ans[0] = i;
                ans[1] = temp.get(diff);
                return ans;
            }
            temp.put(nums[i],i);
        }
        return ans;
    }
}